lms/static/js/student_account/views/LoginView.js

```diff
@@ -204,26 +211,21 @@
                 ];
                 this.clearPasswordResetSuccess();
 
-            /* If we've gotten a 403 error, it means that we've successfully
-             * authenticated with a third-party provider, but we haven't
-             * linked the account to an EdX account.  In this case,
-             * we need to prompt the user to enter a little more information
-             * to complete the registration process.
+            /* If the user successfully authenticated with a third-party provider, but they haven't
+             * linked the account to an Open edX account, instruct the user on how to link the accounts.
```

Comment:
Nit: `linked the account to an Open edX account` -> `linked the accounts`

Rationale: It may be unclear what an `Open edX account` is - it may also mean something else in the future.

Reply:
Updated.